scss/_navbar.scss

```diff
@@ -32,6 +32,31 @@
     align-items: center;
     justify-content: space-between;
   }
+
+  .dropdown-menu {
+    position: static;
+    float: none;
+    margin-top: 0;
+    border: 0;
+    border-radius: 0;
+  }
+
+  @each $color, $value in $theme-colors {
+    &.bg-#{$color} .dropdown-menu {
```

Comment:
Second: this is changing the design. the drop menus would follow the color of the menu. I think I like it, but it might be a visual breaking change.